trompace/exceptions.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, object):
         super().__init__("{} not found in configuration file".format(object))
 
 
-class VaueNotFound(Exception):
```

Comment:
could you make a separate PR for this so that we can get the fixed merged, and continue with the discussion on the date stuff?

Reply:
Done, see [pull request 23](https://github.com/trompamusic/trompa-ce-client/pull/23)